lib/cucumber/cli/configuration.js

```diff
@@ -11,7 +11,8 @@ function Configuration(options, args) {
       var filename = path.basename(arg);
       if (filename[0] === '@') {
         var content = fs.readFileSync(arg, 'utf8');
-        unexpandedFeaturePaths = unexpandedFeaturePaths.concat(content.split('\n'));
+        var paths = content.split('\n').map(_.trim).filter(_.identity);
```

Comment:
I'd prefer we use `_.compact` instead of `filter(_.identify)`. Could also use lodash's `_.chain` in order to keep it as readable as it if now.